src/sonic-yang-models/yang-models/sonic-vrf.yang

```diff
@@ -0,0 +1,49 @@
+module sonic-vrf {
+	namespace "http://github.com/Azure/sonic-vrf";
+	prefix vrf;
+	
+	import sonic-extension {
+		prefix sonic-ext;
+	}
+
+	organization
+		"SONiC";
+
+	contact
+		"SONiC";
+
+	description
+		"SONIC VRF";
+
+	revision 2021-03-30 {
+		description
+			"Initial revision.";
+	}
+
+	container sonic-vrf {
+		container VRF {
+			description "Vrf configuration.";			 
+
+			list VRF_LIST {
+				key "vrf_name";
+
+				leaf vrf_name {
```

Comment:
I think we are using 'name' for all such fields

Reply:
Addressed